active/DensityStyle/UseCompactResources.md

```diff
@@ -0,0 +1,47 @@
+# Background
+Compact feature is described in [Suggestion: Add Compact Dictionary to WinUI 2.1](https://github.com/Microsoft/microsoft-ui-xaml/issues/98).
+This provides API to enable Compact in application level.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+UseCompactResources property is added XamlControlsResources to allow customer to enable Compact.
+
+# Examples
+
+```
+    runtimeclass XamlControlsResources : Windows.UI.Xaml.ResourceDictionary
+    {
+        ...
+        {
+            Boolean UseCompactResources{ get; set; };
+            static Windows.UI.Xaml.DependencyProperty UseCompactResourcesProperty{ get; };

```

Comment:
Do we need this to be a DP?

Reply:
@jevansaks  As I know, we always paired them in idl.

Reply:
Mike's asking if this property needs to be backed by a DP. I don't think so, but our general policy is that properties on controls are dependency properties unless there's a reason for them not to be.

This isn't a control but is a DO. What should the policy be?